images/golang/Dockerfile.centos

```diff
@@ -4,7 +4,7 @@ FROM codercom/enterprise-base:centos
 USER root
 
 # Install go1.15
```

Comment:
please update or remove this comment too

Reply:
ah yes - good catch.